lib/gitlab/client/projects.rb

```diff
@@ -715,5 +715,74 @@ def project_deploy_tokens(project, options = {})
     def project_languages(project)
       get("/projects/#{url_encode project}/languages")
     end
+
+    # List all project access tokens.
+    #
+    # @example
+    #   Gitlab.project_access_tokens(42)
+    #
+    # @param [Integer, String] project The ID or path of a project.
+    # @option options [String] :state Limit by active/inactive state. Optional.
+    #
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def project_access_tokens(project, options = {})
+      get("/projects/#{url_encode project}/access_tokens", query: options)
+    end
+
+    # Get a specific project access token.
+    #
+    # @example
+    #   Gitlab.project_access_token(42, 1234)
+    #
+    # @param [Integer, String] project The ID or path of a project.
+    # @param [Integer] token_id The ID of the project access token.
+    #
+    # @return [Gitlab::ObjectifiedHash] Information about the specified project access token.
+    def project_access_token(project, token_id, options = {})
```

Comment:
Why do we need the `options` argument here?

Reply:
@NARKOZ You're right. That is an oversight. Fixed.